wp-content/themes/core/pcss/components/_dialog.pcss

```diff
@@ -0,0 +1,87 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog[aria-hidden='true'] {
+	display: none;
+}
+
+.c-dialog__overlay {
+	background-color: var(--color-goethite);
+	bottom: 0;
+	color: var(--color-white);
+	left: 0;
+	overflow: scroll;
+	position: fixed;
+	right: 0;
+	top: 0;
+	width: 100%;
+	z-index: 2;
```

Comment:
Have we done any sort of z-indexery as vars, might be worth spinning something up so we can track our usage a bit (can probably wait for later, making a note).